peerpod-ctrl/README.md

```diff
@@ -1,39 +1,26 @@
 # peerpod-ctrl
-The PeerPod CR is used to track peer-pods cloud-provider resources, it specifies the InstanceID and its CloudProvider and it’s owned by the Pod that represents the PeerPod
+The PeerPod CR is used to track peer-pods cloud-provider resources, it specifies the InstanceID and its CloudProvider. PeerPod objects are owned by the matching (peer)Pod object.
 The PeerPod controller is watching PeerPod events and deleting hanging resources that were not deleted by the cloud-api-adaptor at Pod deletion time.
 
 ## Description
 ### Creation time:
-With every successful peer-pod VM creation cloud-api-adaptor will create also a PeePod CR (predefined by the operator) which contains the cloud provider and its VM instance id.
+With every successful peer-pod VM creation cloud-api-adaptor will create a PeePod CR (predefined by the operator) which contains the cloud provider and its VM instance id.
 
 ### Owner references:
-The PeerPod CR is owned by the original Pod object that represents the peer-pod, upon pod deletion background cascading deletion getting into action which means the Pod will be deleted first and the GC will handle the owned PeerPod CR later.
+The PeerPod CR is owned by the original Pod object that represents the peer-pod, upon (peer)Pod deletion [background cascading deletion](https://kubernetes.io/docs/concepts/architecture/garbage-collection/#background-deletion) is getting into action, hence, the Pod will be deleted first and the GC will handle the owned PeerPod CR later.
 
 ### Deletion time:
-* Normal case: When remote hypervisor will get the stopVM request (upon pod deletion) it will delete the podVM instance and if successful it will remove the finalizer attached to the owned PeerPod object so it can then be cleaned by the GC
-* Failure case: In case for some reason cloud-api-adaptor doesn’t honor the delete request of it fails to delete it, the finalizer is not removed and then when peer-pods controller will recognize deletion that the GC is trying to perform for the owned PeerPod object and that it still has the finalizer it will comprehend it needs to preform the deletion of podVM resource based on the PeerPod CR fields
+* Normal case: When remote hypervisor will get the stopVM request (upon (peer)Pod deletion) it will delete the podVM instance and if it succeeds it will remove the finalizer attached to the owned PeerPod object so it can then be cleaned by the GC
```

Comment:
```suggestion
Normal case: When remote hypervisor will get the stopVM request (upon Pod deletion) it will delete the pod VM instance and if it succeeds it will remove the finalizer attached to the owned PeerPod object so it can then be cleaned by the GC.
```